src/Microsoft.DotNet.Build.Tasks.Packaging/src/Generations.cs

```diff
@@ -185,29 +195,29 @@ public Version DetermineGenerationFromSeeds(string assemblyName, Version assembl
 
             if (lowerGeneration == null && upperGeneration == null)
             {
-                log.LogMessage(LogImportance.Low, "Assembly {0} is not tracked by generations.", assemblyName);
+                log.LogMessage(LogImportance.Low, $"Assembly {assemblyName} is not tracked by generations.");
```

Comment:
C# 6.0 features :) Hopefully that doesn't break anything, I don't think it will.